improve serverless dms doc (#15793)
pingcap/docs: tidb-cloud/migrate-from-mysql-using-aws-dms.md

## See also

> - If you want a more general reference on how to connect AWS DMS to TiDB Serverless or TiDB Dedicated, see [Connect AWS DMS to TiDB Cloud clusters](/tidb-cloud/tidb-cloud-connect-aws-dms.md).

Reviewer: ```suggestion
- If you want to learn more about how to connect AWS DMS to TiDB Serverless or TiDB Dedicated, see [Connect AWS DMS to TiDB Cloud clusters](/tidb-cloud/tidb-cloud-connect-aws-dms.md).
```
